internal/testhelpers/suite/suite.go

```diff
@@ -0,0 +1,50 @@
+package suite
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/stretchr/testify/require"
+	testify "github.com/stretchr/testify/suite"
+)
+
+type tSuite interface {
+	testify.TestingSuite
+}
+
+type Suite struct {
+	testify.Suite
+}
+
+func (suite *Suite) NoError(err error, msgAndArgs ...interface{}) {
+	if err == nil {
+		return
+	}
+
+	errMsg := fmt.Sprintf("All error messages: %s", errs.JoinMessage(err))
```

Comment:
Not sure about this string prefix.

Reply:
I don't think it's necessary. Feel free to drop this and just use the result of JoinMessage.